src/reader/ArticleReader.js

```diff
@@ -90,9 +79,19 @@ export default function ArticleReader({ api, teacherArticleID }) {
     );
   }
 
+  function updateScrollPosition() {
+    var scrollElement = document.getElementById("scrollHolder");
+    var scrollY = scrollElement.scrollTop;
+    var limit = scrollElement.scrollHeight - scrollElement.clientHeight - 450; // 450 represents the feedback + exercise div
```

Comment:
Is there a way of computing this as opposed to hardcoding it? The recommender team is about to add a new box at the bottom. That will change this. I wonder whether putting everything that's after the end of the article in a div with a known id, e.v. #feedback, #bottommatter, #backmatter, ... or similar., and then getting the eleemnt by id and querying for it's height?

Reply:
Good point, I have made it by default the 450 in case the id hasn't loaded yet and avoid errors, but now there is a div wrapper which we can query its height by id:
![image](https://github.com/zeeguu/web/assets/17390076/262cef49-917f-45eb-82d4-69809d7f6471)